src/primitives.ts

```diff
@@ -1128,9 +1009,16 @@ export class PieChart {
       this.hidden_color = genColor();   
   }
 
-  buildPath(mvx, mvy, scale, X, Y) {
+  buildPath(mvx: number, mvy: number, scale: number, X: number, Y: number): void {
+    /**
+     * @param mvx x coordinate of the local frame in the canvas
+     * @param mvy y coordinate of the local frame in the canvas
+     * @param scale ratio applied on data coordinates to manage mouse scrolling
+     * @param X x coordinate of the absolute frame (the canvas' one)
```

Comment:
offset of the drawing zone in the plot_data object 
0 if the piechart is not drawn in a multiplot